core/iwasm/aot/aot_runtime.c

```diff
@@ -4663,3 +4664,70 @@ aot_traverse_gc_rootset(WASMExecEnv *exec_env, void *heap)
     return true;
 }
 #endif /* end of WASM_ENABLE_GC != 0 */
+
+char *
+const_str_set_insert(const uint8 *str, int32 len, AOTModule *module,
```

Comment:
How about renaming to `aot_const_str_set_insert`, there is another `const_str_list_insert` function in wasm_runtime.c, it may make confusing.